### PR TITLE
Add missing column #435

### PR DESCRIPTION
**Pull request by robkooper**
@bcow this should add the missing column

### CODE REVIEW

File: db/migrate/20161005181021_add_score_benchmarks_ensembles_scores.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class AddScoreBenchmarksEnsemblesScores < ActiveRecord::Migration
+  def up
+    add_column :benchmarks_ensembles_scores, :score, :text,  :null => false
```
Comment (gsrohde) [inline db/migrate/20161005181021_add_score_benchmarks_ensembles_scores.rb]: @robkooper This really should be textual and not numeric?

**Comment by robkooper** [comment]
This is what @bcow said to me.

File: db/migrate/20161005181021_add_score_benchmarks_ensembles_scores.rb (hunk 1 above)
Comment (bcow) [inline db/migrate/20161005181021_add_score_benchmarks_ensembles_scores.rb]: Yes - many "scores" will not be numeric

**Comment by dlebauer** [comment]
Betsy can you give examples?

**Comment by gsrohde** [comment]
@dlebauer Did Betsy ever answer this to your satisfaction?

**Comment by dlebauer** [comment]
@gsrohde no, I still don't understand what scores would be non-numeric, but you can go ahead and merge / release this.

**Comment by bcow** [comment]
@dlebauer Sorry I missed your response - this is part of @mdietze's original plan. Not all benchmarking functions will produce number scores - for example, functions that produce figures will have the path to the figure file in the scores field. Another example is a test that produces ordinal / categorical results.

**Pull request merged**